pymc/tests/backends/test_arviz.py

```diff
@@ -617,6 +617,27 @@ def test_variable_dimension_name_collision(self):
                 var = at.as_tensor([1, 2, 3])
                 pmodel.register_rv(var, name="time", dims=("time",))
 
+    def test_include_transformed(self):
+        with pm.Model():
+            pm.Uniform("p", 0, 1)
+
+            # First check that the default is to exclude the transformed variables
+            inference_data = pm.sample(
+                500,
+                chains=2,
+                return_inferencedata=True,
+            )
+            assert "p_interval__" not in inference_data.posterior
+
+            # Now check that they are included when requested
+            inference_data = pm.sample(
+                500,
+                chains=2,
+                return_inferencedata=True,
+                idata_kwargs={"include_transformed": True},
+            )
```

Comment:
Just to speed up the test (by a lot)

```suggestion
            inference_data = pm.sample(
                tune=5,
                draws=7,
                chains=2,
                cores=1,
                step=pm.Metropolis(),
            )
            assert "p_interval__" not in inference_data.posterior

            # Now check that they are included when requested
            inference_data = pm.sample(
                tune=5,
                draws=7,
                chains=2,
                cores=1,
                step=pm.Metropolis(),
                idata_kwargs={"include_transformed": True},
            )
```